AppInspector/rules/default/data_types/media.json

```diff
@@ -3,28 +3,20 @@
     "name": "Data: Audio File",
     "id": "AI015600",
     "description": "Data: Audio File",
-    "tags": [
-      "Data.Media.Audio"
-    ],
+    "tags":[ "Data.Media.Audio" ],
     "severity": "moderate",
     "patterns": [
       {
         "pattern": "\\.(wma|wmv|asf|vp9|cr2|wav|mpeg|gsm|ogg|au|aiff|vox|aac|msv|dvf|flac|aifc|amz|atrac|m4a|m4p)",
```

Comment:
Add a `\b` at the end so we don't catch `.auz`, etc.

Reply:
\b is only whitespace so that would not match 

`var filename = "myfile.wma";`

Reply:
mistook `\b` for `\s`